### PR TITLE
Survey template

### PR DESCRIPTION
**Pull request by GalMunGral**

### CODE REVIEW

**Comment by GalMunGral** [comment]
-  Do we want to use Angular Forms (https://angular.io/start/forms)?
- A bug: When I click on a new radio button group the highlight on the previous one disappears.

File: src/app/create-survey/create-survey.component.ts [hunk 1]
```diff
@@ -22,15 +22,14 @@ export class CreateSurveyComponent implements OnInit {
...
-  mcSelection: unknown;
-  mcSelection2: unknown;
   currentProfessor: Professor;
-
-  courseOptions: object[];
+  courseOptions: object[] = [];
   courseSelection: string;
+  surveyDataLoaded: boolean = false;
+  surveyQuestions: { [option: string]: Survey } = {};
+  templateOptions = [{ name:'Default' }, { name: 'CTL' }];
```
Comment (MarcusWilder) [inline src/app/create-survey/create-survey.component.ts]: Is there a particular reason you need an array of OBJECTS here?

File: src/app/create-survey/create-survey.component.html [hunk 2]
```diff
@@ -4,12 +4,17 @@
     <ul>
         <li><a href="index.html" class="active"><p style="margin-top: 185px;"></p></a></li>
     </ul>
-<app-card
-    icon="survey"
-    title="Create Survey"
-></app-card>
```
[the remark this answers is not in the document]
Comment (GalMunGral) [reply src/app/create-survey/create-survey.component.html]: Sure

File: src/app/professor-dashboard/professor-dashboard.component.html [hunk 3]
```diff
@@ -12,7 +12,10 @@
 <app-card
     icon="survey"
     title="Active Surveys"
-    [accordionButtons]="accordionButtons"
-    [accordionTabs]="accordionTabs"
     [cardButtons]="cardButtons"
-></app-card>
+>
+    <app-accordion card-body
```
[the remark this answers is not in the document]
Comment (GalMunGral) [reply src/app/professor-dashboard/professor-dashboard.component.html]: Oops, this should be 'body'.

File: src/app/test/test.component.html [hunk 4]
```diff
@@ -52,8 +52,11 @@
   <app-card
     icon="survey"
     title="Active Surveys"
-    [accordionButtons]="accordionButtons"
-    [accordionTabs]="accordionTabs"
     [cardButtons]="cardButtons"
-  ></app-card>
+  >
+    <app-accordion card-body
```
[the remark this answers is not in the document]
Comment (GalMunGral) [reply src/app/test/test.component.html]: Same, should be 'body'.

File: src/app/create-survey/create-survey.component.ts (hunk 1 above)
Comment (GalMunGral) [reply src/app/create-survey/create-survey.component.ts]: I tried an array of strings and it threw errors. It looks like the pick list is looking for objects with a 'name' key.

**Pull request merged**